### PR TITLE
chore: Refine Wallet UI - Update dropdown, badges, and text.

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**

- Remove `<Badge />` from Wallet docs as it creates confusion because it is missing the schemaID.
- Update all Wallet dropdown SVGs to render 16x16.
- Update text to Claim Basename (if no Basename), otherwise "Profile".
- Update Wallet dropdown text from `Go to Wallet Dashboard` to `Wallet`.
- Vertically align wallet images

<table>
  <tr>
    <th>Before</th>
    <th>After</th>
  </tr>
  <tr>
    <td>

Screenshot 2024-08-06 at 2 44 52 PM

</td>
    <td>

Screenshot 2024-08-06 at 5 12 26 PM

   </td>
  </tr>
</table>

Screenshot 2024-08-06 at 5 12 45 PM

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: site/docs/pages/wallet/wallet.mdx [hunk 1]
```diff
@@ -84,7 +81,6 @@ export function WalletComponents() {
       <Identity className="px-4 pt-3 pb-2" hasCopyAddressOnClick>
         <Avatar />
         <Name>
-          <Badge />
```
Comment (Zizzamia) [inline site/docs/pages/wallet/wallet.mdx:87]: Something is missing here? :)
I let you guess what's missing

Comment (cpcramer) [reply site/docs/pages/wallet/wallet.mdx:87]: Hmm do we want it displayed as `<Name />`

Comment (Zizzamia) [reply site/docs/pages/wallet/wallet.mdx:87]: yup :)

Comment (cpcramer) [reply site/docs/pages/wallet/wallet.mdx:87]: haha sweet, that's the only thing I could think of 😆

File: src/internal/svg/disconnectSvg.tsx [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
+    height="16"
+    viewBox="0 0 16 16"
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
   >
     <path
-      d="M10.9735 2.69696L10.9735 4.52272L5.49622 4.52272L5.49622 15.4773L10.9735 15.4773L10.9735 17.303L3.67046 17.303L3.67046 2.69696L10.9735 2.69696Z"
-      className={fill.defaultReverse}
+      d="M8.77876 2.15757L8.77876 3.61818L4.39698 3.61818L4.39698 12.3818L8.77876 12.3818L8.77876 13.8424L2.93637 13.8424L2.93637 2.15757L8.77876 2.15757Z"
+      fill="#0A0B0D"
```
Comment (Zizzamia) [inline src/internal/svg/disconnectSvg.tsx:13]: Can you check with @kyhyco in making sure we still support the theming here.

Comment (cpcramer) [reply src/internal/svg/disconnectSvg.tsx:13]: Yeah, after the most recent rebase I can't get my BaseNames svg to render

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**